tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+from pyfakefs.fake_filesystem_unittest import Patcher
+from pytest import fixture
+
+
+@fixture(autouse=True)
+def global_fake_fs() -> None:
+    with Patcher():
+        yield
```

Comment:
To be honest, I preferred explicitly adding fs to the relevant tests. That way there's no hidden functionality, you know which tests are hitting a fake file system and which do not.
But up to you, I'm not too worried about it.

Reply:
This is required because now any authentication leaves token cache artifact file, which also causes tests to fail

Reply:
We would need to add fake fs to almost every integration test and to a lot of unit tests